aws-ebs-csi-driver/templates/rolebinding-snapshot-controller-leaderelection.yaml

```diff
@@ -13,7 +13,7 @@ subjects:
     namespace: kube-system
 roleRef:
   kind: Role
-  name: snapshot-controller-leaderelection
+  name: ebs-snapshot-controller-leaderelection
```

Comment:
I mentioned this briefly in the commit message / PR description, but I'm unsure how this repository's snapshot controller was working if the snapshot controller from the external-snapshotter repository was installed into a namespace other than kube-system. In that case, the role previously referenced here would not have existed. Are we sure that this role / role binding are needed?